apps/labs/posts/numpy-accessibility-guidelines.md

```diff
@@ -0,0 +1,187 @@
+---
+title: 'Making Numpy Accessible: Guidelines and Tools'
+published: September 06, 2021
+author: mars-lee
+description: 'In response to this, a small team started working on making NumPy more accessible, specifically its website and documentation. They weren’t experts in accessible technology. In fact, they feared that they didn’t know enough. Yet they strongly believed that accessible technology is a right every person should access and that NumPy could be accessible too.'
+category: [Access-centered]
+featuredImage:
+  src: /posts/numpy-accessibility-guidelines/feature.png
+  alt: 'A large eye is placed over two bar graphs. Two silhouettes of heads are also
+overlaid the bar graphs.'
+hero:
+  imageSrc: /posts/numpy-accessibility-guidelines/hero.png
+  imageAlt: 'A large eye is placed over two bar graphs. Two silhouettes of heads are also
+overlaid the bar graphs.'
+---
+
+_Header illustration by author, Mars Lee_
+
+Numpy is now foundational to Python scientific computing. Our efforts reach
+millions of developers each month. As our user base grows, we recognize that we
+are neglecting the disabled community by not having our website and
+documentation up to modern accessibility standards.
+
+![The Venn diagram shows 2 intersecting circles, one labeled '10 million NumPy
+users' and the other labeled '15% of human population is disabled'. The area of
+intersection is labeled '1,500,000 disabled NumPy
+users.](/posts/numpy-accessibility-guidelines/venn_diagram.png)
+
+As NumPy is early in the dependency chain, it has a huge install base. It is
+estimated that there are 10 million users of NumPy. Around 15% of the general
+human population disabled. ([World Health
+Organization](https://www.who.int/disabilities/world_report/2011/report.pdf)).
```

Comment:
@MarsBarLee the WHO link is broken. I found an alternative that could work. Can you please check it's correct?
```suggestion
Organization](https://www.who.int/publications/i/item/9789241564182)).
```